### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by sw990**
Worked on home page and began on alliance selection page but layout for that is not created yet.

image

### CODE REVIEW

**Comment by NormalDuck** [comment]
Be sure to check the warnings for `files changed` section. You can ignore some of it.
Also there is merge conflicts!!!!
image

File: .vscode/settings.json
Comment (NormalDuck) [inline .vscode/settings.json:1]: Why did you remove eslint as your default formatter for typescript-react?

Comment (sw990) [reply .vscode/settings.json:1]: I was formatting my code, and it didn't work so I ended up switching to prettier

Comment (johnylamw) [reply .vscode/settings.json:1]: Yeah.. WHY?!?!?!

**Review by johnylamw — APPROVED** [review]
Looks good. BUT WHY?!
